### PR TITLE
Standardize deprecation

### PR DESCRIPTION
**Pull request by nemchik**

### CODE REVIEW

File: roles/generate-jenkins/templates/Jenkinsfile.j2 [hunk 1]
```diff
@@ -481,15 +482,22 @@ pipeline {
...
+                  git rm unraid/${CONTAINER_NAME}.xml || :
                 else
                   cp ${TEMPDIR}/docker-${CONTAINER_NAME}/.jenkins-external/${CONTAINER_NAME}.xml ${TEMPDIR}/unraid/templates/unraid/
-                  cd ${TEMPDIR}/unraid/templates/
                   git add unraid/${CONTAINER_NAME}.xml
-                  git commit -m 'Bot Updating Unraid Template'
-                  git push https://LinuxServer-CI:${GITHUB_TOKEN}@github.com/linuxserver/templates.git --all
                 fi
+                git commit -m 'Bot Updating Unraid Template'
+                git push https://LinuxServer-CI:${GITHUB_TOKEN}@github.com/linuxserver/templates.git --all
```
[the remark this answers is not in the document]
Comment (nemchik) [reply roles/generate-jenkins/templates/Jenkinsfile.j2:501]: Fixed in https://github.com/linuxserver/docker-jenkins-builder/pull/101/commits/c82c0b4050bf36fb3681526fd59dcdc441dfb92d

File: roles/generate-jenkins/templates/DOCUMENTATION.j2 [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
+{% if project_deprecation_status %}
+# DEPRECATION NOTICE
+
+This image is deprecated. It will no longer receive automated updates.
+{% if project_deprecation_message is defined %}{{ project_deprecation_message }}{% endif %}
+
+{% endif %}
 # [{{ lsio_project_name_short }}/{{ project_name }}]({{ lsio_github_url }}/{{ project_repo_name }})
+<!-- DO NOT EDIT THIS FILE MANUALLY  -->
+<!-- Please read the {{ lsio_github_asset }}/.github/CONTRIBUTING.md -->
```
Comment (Roxedus) [inline roles/generate-jenkins/templates/DOCUMENTATION.j2:13]: Do we want this here? adding it under the metadata makes sense in my eyes

Comment (nemchik) [reply roles/generate-jenkins/templates/DOCUMENTATION.j2]: Resolved in https://github.com/linuxserver/docker-jenkins-builder/pull/101/commits/c5550f163cc6d2814c5cad2e1b895a1d7d8226e3

File: roles/generate-jenkins/templates/Jenkinsfile.j2 [hunk 3]
```diff
@@ -451,6 +464,7 @@ pipeline {
                 cd ${TEMPDIR}/docker-${CONTAINER_NAME}
                 mkdir -p ${TEMPDIR}/repo/${LS_REPO}/.github/workflows
                 mkdir -p ${TEMPDIR}/repo/${LS_REPO}/.github/ISSUE_TEMPLATE
+                mkdir -p ${TEMPDIR}/repo/${LS_REPO}/root/etc/cont-init.d
```
Comment (aptalca) [inline roles/generate-jenkins/templates/Jenkinsfile.j2:467]: Should this be in an if statement based on deprecation?

Comment (nemchik) [reply roles/generate-jenkins/templates/Jenkinsfile.j2:468]: Resolved in https://github.com/linuxserver/docker-jenkins-builder/pull/101/commits/8bd329270200113449a568e4ca3970d97d5b9937

**Pull request merged**